src/main/java/lk/ac/pdn/co328/studentSystem/StudentRegister.java

```diff
@@ -1,81 +1,78 @@
+/*
+E/12/302
+ */
 package lk.ac.pdn.co328.studentSystem;
+
 import java.util.ArrayList;
 
-public class StudentRegister
-{
+public class StudentRegister {
     private ArrayList<Student> studentList = new ArrayList<Student>();
 
+
     // Adds a new student to the system
     public void addStudent(Student st) throws Exception {
-        for (Student student:studentList)
-        {
-            if(student.getId() == st.getId())
-            {
+        for (Student student : studentList) {
+            if (student.getId() == st.getId()) {
                 throw new Exception("StudentID already exists in the register");
             }
         }
         studentList.add(st);
     }
 
     // Remove a student from the system
-    public void removeStudent(int regNo)
-    {
-        for (int i = 0; i<studentList.size(); i++)
-        {
-            if(studentList.get(i).getId() == regNo)
-            {
+    public void removeStudent(int regNo) {
+        for (int i = 0; i < studentList.size(); i++) {
+            if (studentList.get(i).getId() == regNo) {
                 studentList.remove(i);
             }
         }
     }
 
     //Finds the student with the given registration number
-    public Student findStudent(int regNo)
-    {
-        for (int i = 0; i<studentList.size(); i++)
-        {
-            if(studentList.get(i).getId() == regNo)
-            {
-                return studentList.get(i);
+    public Student findStudent(int regNo) {
```

Comment:
No need for use a for loop with iterating to size of the list
but we can use foreach here it is easy in array lists there are many places this error is but i change several of them 
where i could found in a one min scanning of code i'm too lazy to scan through these lines got better things to do bear with me bro